build/Packaging/ScriptableInstaller/NewRelic.Net.Agent.x64.nuspec

```diff
@@ -6,9 +6,9 @@
     <title>New Relic .NET Agent (x64)</title>
     <authors>New Relic</authors>
     <owners>New Relic</owners>
-    <projectUrl>https://newrelic.com/.NET</projectUrl>
+    <projectUrl>https://docs.newrelic.com/docs/apm/agents/net-agent/other-installation/net-agent-install-resources#scriptable-installer</projectUrl>
```

Comment:
Should this be our github repo?

Reply:
I believe this issue has come up before and the answer is that we want to link to our documentation.  For what it's worth, I looked at some other popular NuGet packages in our space and they all link to their own documentation rather than a GitHub repo.